python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -146,6 +176,212 @@ def create_domain(
 
         return domain_urn
 
+    def upsert_athena_database(self, metadata: DatabaseMetadata) -> str:
+        """
+        Define a database. Must belong to a domain
+        """
+        metadata_dict = dict(metadata.__dict__)
+        metadata_dict.pop("version")
+        metadata_dict.pop("owner")
+        metadata_dict.pop("tags")
+
+        name = metadata_dict.pop("name")
+        description = metadata_dict.pop("description")
+
+        database_urn = "urn:li:container:" + "".join(name.split())
```

Comment:
It might be safer to use Datahub's own methods for building URNs. Just makes it easier to keep up to date if they ever change the format. 

I'm also wondering if we need to pass the ENV component of the URN here - so far everything we've created goes in PROD, but I'm not sure if we are going to need to catalogue preprod metadata at any point.